src/devices/Common/FrameworkCompatibilityExtensions.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+#pragma warning disable CS1591 // Missing documentation (these methods are mostly hidden from the user)
+#if NETSTANDARD2_0
```

Comment:
Looks like you still haven't removed this Ifdef and instead conditionally compile the file. Same goes for IsExternalInit. I think that is the last remaining feedback I have and this should be good to go.